code/game/objects/effects/landmarks/corpsespawner.dm

```diff
@@ -15,6 +18,7 @@
 /obj/effect/landmark/corpsespawner
 	name = "Unknown"
 	icon_state = "skullmarker"
+	var/death_type = COCOONED_DEATH //how did victim die? determines if we should burst, headbite them etc. we default to cocoon
```

Comment:
You are not adding variant ? 
Plus autodoc, but the comments are not good anyway